swagger2.0/oic.sec.cred.swagger.json

```diff
@@ -430,8 +430,8 @@
                 "type": "integer"
               },
               "credtype": {
-                "description": "Representation of this credential's type\nCredential Types - Cred type encoded as a bitmask.0 - Empty credential used for testing1 - Symmetric pair-wise key2 - Symmetric group key4 - Asymmetric signing key8 - Asymmetric signing key with certificate16 - PIN or password32 - Asymmetric encryption key.",
-                "maximum": 63,
+                "description": "Representation of this credential's type\nCredential Types - Cred type encoded as a bitmask.0 - Empty credential used for testing\n1 - Symmetric pair-wise key\n2 - Symmetric group key\n4 - Asymmetric signing key\n8 - Asymmetric signing key with certificate\n16 - PIN or password\n32 - Asymmetric encryption key. \n 128 - SSM Client\n256 - SSM Server",
+                "maximum": 256,
```

Comment:
Oh, we added another bit to "sct" so now it's 9 bits long, right?  That means the max value is 0b1 1111 1111 or 511.

Reply:
no, it is just a number..
the highest number is 256.

Reply:
So what value does "sct" take if you support 256 - SSM Server, 1 - Symmetric pair-wise, and 8 - Asymmetric signing key with certificate?

Reply:
I think it's a bit mask.

Reply:
let's ask phil.

Reply:
the resource is an array, each entry is just a single credential.
yes, written up as bit entry, but actually is is just a single number to indicate which cred type that array entry is.

Reply:
Yep, I was thinking of the "sct" Property, not the "credtype".  We need to update the "sct" Property as well which is how the Device exposes which credential types it supports.

Reply:
The maxValue of "credtype" should be 256.  The maxValue of doxm.sct should be 511.